RadixWallet/Features/DappInteractionFeature/Children/PersonaDataPermission/PersonaDataPermissionBox.swift

```diff
@@ -5,12 +5,12 @@ struct PersonaDataPermissionBox: Sendable, FeatureReducer {
 	struct State: Sendable, Hashable, Identifiable {
 		var id: Persona.ID { persona.id }
 		let persona: Persona
-		let requested: P2P.Dapp.Request.PersonaDataRequestItem
-		let responseValidation: P2P.Dapp.Request.RequestValidation
+		let requested: DappToWalletInteractionPersonaDataRequestItem
+		let responseValidation: DappToWalletInteraction.RequestValidation
```

Comment:
Shouldn't this be in Sargon?

Reply:
It does, along with logic from [here](https://github.com/radixdlt/babylon-wallet-ios/blob/9d6e988e81fdc5ad6330b52ecc36388e8420a0ca/RadixWallet/MIGRATE_TO_SARGON/Stage1/Stage1MigrateToSargon%2BSignedAuthChallenge.swift), [here](https://github.com/radixdlt/babylon-wallet-ios/blob/9d6e988e81fdc5ad6330b52ecc36388e8420a0ca/RadixWallet/MIGRATE_TO_SARGON/Stage2/RadixConnect/Stage2MigrateToSargon%2BRadixConnectModels%2BLogic.swift) and [here](https://github.com/radixdlt/babylon-wallet-ios/blob/9d6e988e81fdc5ad6330b52ecc36388e8420a0ca/RadixWallet/Core/SharedModels/P2P/Dapp/Models/WalletInteraction%2BExtensions.swift). We can prioritize them accordingly and add them in the future.